.release-it.json

```diff
@@ -5,8 +5,8 @@
 	"force": false,
 	"pkgFiles": ["package.json"],
 	"increment": "conventional:angular",
-  "beforeChangelogCommand": "npm run generate:changelog",
```

Comment:
Same comment as for Christopher’s latest PR, I think we should try to isolate styling changes in their own ‘chore’ PRa to limit the noise.